cmd/tunasynctl/tunasynctl.go

```diff
@@ -185,7 +185,7 @@ func listJobs(c *cli.Context) error {
 			fmt.Sprintf("Error printing out informations: %s", err.Error()),
 			1)
 	}
-	fmt.Printf(string(b))
+	fmt.Print(string(b))
```

Comment:
Oh, you fixed a possible security vulnerability, great!
Change it to `Println` would be better.